Data Preparation.R

```diff
@@ -0,0 +1,173 @@
+##################################################################
+## Name of File: End of Life All Activity                       ##
+## Original Authors : Federico Centoni/Alice Byers              ##
+## Originally written: November 2018                            ##                      
+## Latest Update Date: 23/07/2019.                              ##
+## Written/run on : R Studio Server                             ##
+## R version:                                                   ##
+## Package version:                                             ##
+## Description: Outputs All Activity file to the patient level  ##
+##              for use in EOL publication                      ##
+##################################################################
+
+
+
+###################################
+## SECTION 1 - HOUSEKEEPING      ##
+###################################
+
+install.packages("readxl")
+install.packages("here")
+install.packages("readr")
+install.packages("dplyr")
+install.packages("tidyr")
+install.packages("odbc")
+install.packages("janitor")
+
+## 1. Load packages ##
+
+library(odbc)      # for accessing SMRA
+library(dplyr)     # functions for 'tidy' data wrangling
+library(haven)     # read and write spss files
+library(lubridate) # functions for use with dates
+library(magrittr)  # for %<>% operator
+library(tidyverse)
+library(janitor)   
+library(tictoc)
+
+
+## 2. Define dates ##
+
+extract_date <- "_extracted_22072019"
```

Comment:
What is this used for? This shouldn't be hardcoded like this - if it is needed for a filename that is being saved out, include the "_extracted_" as part of the filename and then insert the date using a date object you have defined below.